XenAdmin/Commands/PauseVMCommand.cs

```diff
@@ -0,0 +1,226 @@
+/* Copyright (c) Citrix Systems, Inc. 

```

Comment:
Does creating a new file need to keep the Citrix copyright?

Reply:
Yes good question, I don't know, it is based on existing files so ...
But I have nearly the same question for icons, a quick workaround to not have the same Icon for Pause and Suspend would be to simply slightly recolor one of the existing blue one, for example orange or purple or what ever, but would this be OK to "modify" existing icons?

Reply:
I'd use something like in this answer: https://softwareengineering.stackexchange.com/a/158011

```
Original work Copyright (c) Citrix Systems, Inc.
Modified work Copyright XCP-ng project
```

Only for files where we modify or duplicate portions of code written by Citrix... For other files a copyright notice is not mandatory to ensure that copyright laws apply.

Reply:
Note that although there's no legal entity called "XCP-ng project", the copyright notice does not have to use a name that has a legal existence (for the same reason that makes the copyright notice not mandatory at all, provided there's a way to prove who wrote what, like git provides). That's why I suggest "XCP-ng project" so that anyone from the community could feel to be covered by that label. Or "XCP-ng community".

Reply:
Forgot to answer about the icons: they are covered by the same license as the code, unless there's a file that says otherwise (in which case we'll need to check that we are allowed to use them at all!). So unless there's a different license for them that would restrict derivative work, you can modify them at will.

Reply:
Thanks for the details! I will change  the text and icons as next then.